si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+    class `ScoreDocumentsProxyModel`:
+        def `_alphanum_key`:
+            ([0-9]+): false
+    class `ScoreDocumentsTableModel`:
+        def `simplify`:
+            \n\t\r: false
+            ' ': false
+    class `OWScoreDocuments`:
+        Score Documents: Oceni dokumente
+        icons/ScoreDocuments.svg: false
+        score documents: score documents, ocenjevanje, vrednotenje, točkovanje
+        class `Inputs`:
+            Corpus: Korpus
+            Words: Besede
+        class `Outputs`:
+            Selected documents: Izbrani dokumenti
+            Corpus: Korpus
+        class `Warning`:
+            Use Preprocess Text to normalize corpus.: Uporabi gradnik Predprocesiranje besedila da normiraš korpus.
+            {}: false
+        class `Error`:
+            {}: false
+        def `_setup_control_area`:
+            Word Scoring Methods: Metode za ocenjevanje besed
+            Aggregation: Izračunaj
+            aggregation: false
+            Select Documents: Izberi dokumente
+            n_selected: false
+            auto_commit: false
+        def `_setup_main_area`:
+            Filter...: Filtriraj...
+            Document: Dokument
+        def `_get_word_attribute`:
+            type: false
+            words: false
+        def `_fill_table`:
+            Document: Dokument
+        def `on_partial_result`:
+            '{SCORING_METHODS[sc_method][0]} failed: {scores}': {SCORING_METHODS[sc_method][0]} neuspešno: {scores}
+    __main__: false
+    book-excerpts: false
+    house: false
+    doctor: false
+    boy: false
+    way: false
+    Rum: false
+widgets/owsemanticviewer.py:
+    "
+<!doctype html>
+<html>
+<head>
+<script type=""text/javascript"" src=""resources/jquery-3.1.1.min.js"">
+</script>
+<script type=""text/javascript"" src=""resources/jquery.mark.min.js"">
+</script>
+<script type=""text/javascript"" src=""resources/highlighter.js"">
+</script>
+<meta charset='utf-8'>
+<style>
+
+mark {{ 
+    background: #FFCD28;
+}}
+body {{
+    font-family: Helvetica;
+    font-size: 10pt;
+}}
+hr {{
+    border: none;
+    height: 1px;
+    background-color: #000;
+
+}}
+
+</style>
+</head>
+<body>
+{}
+</body>
+</html>
+": false
+    def `run`:
+        Calculating...: Računanje...
+    class `DocumentsModel`:
+        def `_argsortData`:
+            mergesort: false
+    class `DisplayDocument`:
+        Document: Dokument
+        Section: Razdelek
+        Sentence: Stavek
+        <mark data-markjs='true'>: false
+        </mark>: false
+        {START_TAG}{{}}{END_TAG}: false
+        \n: false
+        ...: false
+        def `__call__`:
+            {self.SECTION_SEP}+: false
+            ' ': false
+        def `_replace_sentences`:
+            def `replace_unmatched`:
+                ' ': false
+    class `OWSemanticViewer`:
+        Semantic Viewer: Semantični pregledovalnik
+        Infers characteristic words from the input corpus.: Iz vhodnega korpusa izpelje karakteristične besede.
```

Comment:
```suggestion
        Infers characteristic words from the input corpus.: false
```
This description is wrong in Orange. I will fix it, and then we can translate it. I am just setting it to false temporarily.